tests/functional/test_lambda_trigger_events.py

```diff
@@ -43,6 +43,18 @@ def load_event(file_name: str) -> dict:
         return json.load(fp)
 
 
+def test_dict_wrapper_equals():
+    class DataClassSample(DictWrapper):
+        @property
+        def message(self) -> str:
+            return self.get("message")
+
+    assert DataClassSample({"message": "foo1"}) == DataClassSample({"message": "foo1"})
+    assert DataClassSample({"message": "foo1"}) != DataClassSample({"message": "foo2"})
+    assert DataClassSample({"message": "foo1"}) is not object()
+    assert object() is not DataClassSample({"message": "foo1"})
+
```

Comment:
Could you make this into a literal dict object and reuse it?

This will esse changing that one place only if we must in the future :)

Reply:
Sure, like this you mean: 0ab2786?